suzieq/gui/stlit/guiutils.py

```diff
@@ -1,13 +1,13 @@
-from importlib.util import find_spec
+import base64
 import os
 from enum import Enum
-import base64
+from importlib.util import find_spec
```

Comment:
I'm confused. Did you add these? We shouldn't be needing them, IIRC.

Reply:
I didn't added them. It is only import reordering.
base64 is used by the image and find_spec is used by the function `get_image_dir`